documents/protocols/code guide.md

```diff
@@ -1,10 +1,10 @@
 # The Definitive Vyxal Code Style Guide
-_Fourteenth revision_
+_Sixteenth revision_
```

Comment:
Do we really need to keep track of these revisions explicitly? You can just look back at the file's history in git

Reply:
It's mostly for posterity.

Reply:
Git has posterity built-in? (PS: please don't "Resolve conversation" if discussion is clearly ongoing)

Reply:
I like having an explicit revision count.